mlflow/projects/__init__.py

```diff
@@ -268,8 +253,7 @@ def _get_conda_bin_executable(executable_name):
     return executable_name
 
 
-def _maybe_create_conda_env(conda_env_path):
-    conda_env = _get_conda_env_name(conda_env_path)
+def _maybe_create_conda_env(project_dir):
```

Comment:
Can we add documentation to this method as to what the `maybe` means (i.e., if not created before). Perhaps if this were `_get_or_create_conda_env`, and it returned the conda environment name, which would be passed into `_get_entry_point_command` instead of `use_conda`?